examples/demo/schema/lookup/lookup_schema.sql

```diff
@@ -2,4 +2,4 @@ create table user_seq(id int, next_id bigint, cache bigint, primary key(id)) com
 insert into user_seq(id, next_id, cache) values(0, 1, 3);
 create table music_seq(id int, next_id bigint, cache bigint, primary key(id)) comment 'vitess_sequence';
 insert into music_seq(id, next_id, cache) values(0, 1, 2);
-create table name_user_idx(name varchar(128), user_id bigint, primary key(name, user_id));
+create table name_user_idx(name varchar(128), user_id binary(8), primary key(name, user_id));
```

Comment:
Change to `name_idx` and `keyspace_id`

Reply:
Not sure I follow this rename. Do you mean this? 

```
create table name_keyspace_idx(name varchar(128), keyspace_id binary(8), primary key(name, keyspace_id));
```

Reply:
`name_keyspace_idx` would also work.